lib/webResource.ts

```diff
@@ -68,21 +69,23 @@ export class WebResource {
     body?: any,
     query?: { [key: string]: any; },
     headers?: { [key: string]: any; } | HttpHeaders,
-    rawResponse = false,
```

Comment:
Changed this because I don't trust the compiler to emit something that works in all environments